some tests take longer to execute and this may cause some tests pass or fail at random

diff --git a/tests/check/meson.build b/tests/check/meson.build
index c2702f3..e9fc4f4 100644
--- a/tests/check/meson.build
+++ b/tests/check/meson.build
@@ -158,7 +158,7 @@ foreach t : base_tests
     env.set('GST_PLUGIN_PATH_1_0', meson.build_root(), pluginsdirs)
     env.set('GST_PLUGIN_SYSTEM_PATH_1_0', '')
     env.set('GST_STATE_IGNORE_ELEMENTS', 'cdio cdparanoiasrc libvisual_ alsasrc alsasink')
-    env.set('CK_DEFAULT_TIMEOUT', '20')
+    env.set('CK_DEFAULT_TIMEOUT', '60')
     env.set('GST_TAG_LICENSE_TRANSLATIONS_DICT', join_paths(gst_tag_dir, 'license-translations.dict'))
     env.set('GST_PLUGIN_LOADING_WHITELIST', 'gstreamer',
         'gst-plugins-base@' + meson.build_root())
@@ -181,7 +181,7 @@ foreach group : [1, 2, 3, 4, 5, 6]
   env.set('GST_PLUGIN_PATH_1_0', meson.build_root(), pluginsdirs)
   env.set('GST_PLUGIN_SYSTEM_PATH_1_0', '')
   env.set('GST_STATE_IGNORE_ELEMENTS', 'cdio cdparanoiasrc libvisual_ alsasrc alsasink')
-  env.set('CK_DEFAULT_TIMEOUT', '20')
+  env.set('CK_DEFAULT_TIMEOUT', '60')
   env.set('GST_TAG_LICENSE_TRANSLATIONS_DICT', join_paths(gst_tag_dir, 'license-translations.dict'))
   env.set('GST_PLUGIN_LOADING_WHITELIST', 'gstreamer',
       'gst-plugins-base@' + meson.build_root())
